packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -109,7 +109,7 @@ export class AttestationService {
     try {
       attesterDuties = await this.provider.validator.getAttesterDuties(epoch, this.publicKeys);
     } catch (e) {
-      this.logger.error(`Failed to obtain attester duty for epoch ${epoch}`, e);
+      this.logger.error(`Failed to obtain attester duty for epoch ${epoch}`, e.message);
```

Comment:
Now that the logger issues are fixed, should we send the entire error object to the logger to print the stack trace?

Reply:
TBH, those stacktraces can be too much on validator side. Maybe if they are only enabled on debug level. But just useless clutter here.